test/file.jl

```diff
@@ -54,19 +54,27 @@ child_eval(code::String) = eval(Meta.parse(readchomp(`$(Base.julia_cmd()) -E $co
     # mktemp without cleanup
     t = child_eval("t = mktemp(cleanup=false)[1]; @assert isfile(t); t")
     @test isfile(t)
-    rm(t, force=true)
+    rm(t)
```

Comment:
This line should not have been changed.
Should it?
If the above test fails then won't this now error?